docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
```

Comment:
I've filed an issue to talk about sorting out this nomenclature:
https://github.com/vigetlabs/microcosm/issues/268

The gist of it is - I think I'm wrong. We should just call actions _actions_, not commands. The thing we get back from `repo.push()` should be called a separate thing. That prevents a breaking change and there's no reason to change existing code.